addons/event/security/ir.model.access.csv

```diff
@@ -26,3 +26,9 @@ access_event_tag,event.tag,model_event_tag,,0,0,0,0
 access_event_tag_registration,event.tag.user,model_event_tag,event.group_event_registration_desk,1,0,0,0
 access_event_tag_user,event.tag.user,model_event_tag,event.group_event_user,1,1,1,0
 access_event_tag_manager,event.tag.manager,model_event_tag,event.group_event_manager,1,1,1,1
+access_event_question_employee,event.question,model_event_question,base.group_user,1,0,0,0
```

Comment:
Do we need to give access to those models to internal users ? They don't have access to event models so I suppose they should not ?

Reply:
I don't really know it was like that before

Reply:
Yes in website_event, but not in base event module :)